main.go

```diff
@@ -369,17 +374,32 @@ func generateCrdDocs(apiExtensionsTag string) error {
 			fmt.Printf("Something went wrong in ReadCRD: %#v\n", err)
 		}
 
-		fmt.Printf("Writing output for CRD %s\n", crd.Name)
+		if contains(crdsToSkip, crd.Name) {
+			fmt.Printf("Skipping CRD %s\n", crd.Name)
+		} else {
+			fmt.Printf("Writing output for CRD %s\n", crd.Name)
 
-		err = WriteCRDDocs(crd, outputFolderPath, apiExtensionsTag)
-		if err != nil {
-			fmt.Printf("Something went wrong in WriteCRDDocs: %#v\n", err)
+			err = WriteCRDDocs(crd, outputFolderPath, apiExtensionsTag)
+			if err != nil {
+				fmt.Printf("Something went wrong in WriteCRDDocs: %#v\n", err)
+			}
 		}
 	}
 
 	return nil
 }
 
+// contains checks whether slice contains the given item.
+func contains(slice []string, item string) bool {
+	set := make(map[string]struct{}, len(slice))
+	for _, s := range slice {
+		set[s] = struct{}{}
```

Comment:
Why is creating a set needed? Can't we just:
```go
if s == item {
    return true
}
```
and just `return false` if the loop is done and the string is not found

Reply:
Yeah, that happens when you copy a popular answer on StackOverflow...